changelog/CHANGES.md

```diff
@@ -0,0 +1,14 @@
+# Master
+
+## Summary
+
+## Upgrade notes
+
+The following changes were made to the API:
+
+ * `ol.events.condition.mouseMove` was replaced by `ol.events.condition.pointerMove`, so your `ol.interaction.Select` that were using it have to be updated. (see [#3281](https://github.com/openlayers/ol3/pull/3281)).
+
+
+## Changes
```

Comment:
I don't think we need the `Changes` section. This one is automatically generated.